var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -178,6 +179,9 @@ def _configure_args(self):
                 '--disable-cub'
             ])
 
+        if '+unified-memory' in self.spec:
```

Comment:
Should this option only be applied if `cuda` is enabled?

Reply:
Oof, yes. Let me address that.

Reply:
Okay, @tldahlgren , I think that should fix that.